index.bs

```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
```

Comment:
I'd probably put this at the *end* of the CSS section rather than the beginning.

Reply:
Done